### ISSUE TITLE
Compiling v1.19.8 fails on openSUSE stemcells (fixed by bumping automake to 1.16.1)

### ISSUE THREAD

**Issue by aduffeck** [cloudfoundry/garden-runc-release#152]
## Description
We are having problems compiling v1.19.8 on the openSUSE stemcell due to some incompatibility with the bundled automake 1.16 package. Upgrading automake to the latest version (1.16.1) fixes that issue.

## Environment

* garden-runc-release version: v1.19.8
* Stemcell version: `splatform/fissile-stemcell-opensuse:42.3-36.g03b4653-30.95`

## Steps to reproduce

Extract the release in the opensuse stemcell (e.g. `splatform/fissile-stemcell-opensuse:42.3-36.g03b4653-30.95`) and try to compile the automake package.

## Logs

Compilation fails with

```
config.status: creating Makefile
config.status: creating pre-inst-env
  GEN      bin/automake
  GEN      bin/aclocal
  GEN      bin/aclocal-1.16
  GEN      bin/automake-1.16
  GEN      t/ax/shell-no-trail-bslash
  GEN      t/ax/cc-no-c-o
  GEN      runtest
  GEN      doc/aclocal.1
  GEN      doc/automake.1
  GEN      lib/Automake/Config.pm
  GEN      doc/aclocal-1.16.1
  GEN      doc/automake-1.16.1
help2man: can't get `--help' info from automake-1.
Try `--no-discard-stderr' if option outputs to stderr
Makefile:3694: recipe for target 'doc/automake-1.16.1' failed
make: *** [doc/automake-1.16.1] Error   
```

## Docker image (if relevant)

https://hub.docker.com/r/splatform/fissile-stemcell-opensuse

**Comment by bikramnehra** [comment]
We have tested the fix proposed by @aduffeck here: https://github.com/bikramnehra/garden-runc-release/tree/v1.19.9-bump-automake. For now we are using a fork which we would like to get rid of as soon as this fix is incorporated.

**Comment by gcapizzi** [comment]
It looks like this has been fixed since `1.19.10` 😂 closing!
